webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -90,7 +90,7 @@ class BookmarkContainer extends React.Component {
                 />
               ))) || <MenuItem disabled> {__('None found')}</MenuItem>)}
           {status === STATUS.ERROR && (
-            <MenuItem key="bookmarks-errors">
+            <MenuItem bsClass="bookmarks-errors" key="bookmarks-errors">
```

Comment:
How is this related?

Reply:
it was, for reducing snapshots and getting only the related part for it, but I have found another way